tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHttpManagerExtensions.cs

```diff
@@ -162,7 +163,8 @@ public static void AddMockHandlerContentNotFound(this MockHttpManager httpManage
                 });
         }
 
-        public static MockHttpMessageHandler AddMockHandlerSuccessfulClientCredentialTokenResponseMessage(this MockHttpManager httpManager)
+        public static MockHttpMessageHandler AddMockHandlerSuccessfulClientCredentialTokenResponseMessage(
+            this MockHttpManager httpManager)
```

Comment:
Are we intending to follow this pattern going forward?

Also, there seem to be a few methods below that are not following this

Reply:
@trwalke, do you mean newline after the ( ?

---
In reply to: [401286305](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1735#discussion_r401286305) [](ancestors = 401286305)

Reply:
@henrik-me yes

Reply:
There are no hard limits on the max length of a method / line, but generally I'd say that code should be readable without having to scroll horizontally (on a typical 1080p resolution, with VS having Test Explorer and Solution Explorer open).